tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/Search.cs

```diff
@@ -130,10 +143,10 @@ public SearchResponseVM GetMatters(SearchRequestVM searchRequestVM)
                     managedProperties.Add(searchSettings.ManagedPropertyDescription);
                     managedProperties.Add(searchSettings.ManagedPropertySiteName);
                     managedProperties.Add(searchSettings.ManagedPropertyLastModifiedTime);
-                    var managedColumns = configuration.GetSection("ContentTypes").GetSection("ManagedColumns").GetChildren();
+                    var managedColumns = configuration.GetSection("ContentTypes").GetSection("ManagedStampedColumns").GetChildren();
                     foreach (var key in managedColumns)
-                    {
-                        managedProperties.Add(searchSettings.ManagedPropertyExtension + key.Value.Replace("LPC", ""));
+                    {          
```

Comment:
When getting data from SPO search, used ManagedStampedColumns section and not ManagedColumns section